widgetssdk/src/main/java/com/glia/widgets/chat/adapter/ChatAdapter.java

```diff
@@ -59,6 +62,8 @@ public boolean areContentsTheSame(@NonNull ChatItem oldItem, @NonNull ChatItem n
                 return oldItem.equals(newItem);
             } else if (oldItem instanceof VisitorAttachmentItem && newItem instanceof VisitorAttachmentItem) {
                 return oldItem.equals(newItem);
+            } else if (oldItem instanceof CustomMessageItem && newItem instanceof CustomMessageItem) {
+                return oldItem.equals(newItem);
             } else {
                 return false;
             }
```

Comment:
I know this is not related to your changes but could you please change this bunch of if-else to return `oldItem.equals(newItem);` because as I understand if instances are different it will return false

Reply:
I guess in this way it can return true only for specific values. Let's postponed with this changes.